packages/@okta/vuepress-site/docs/guides/anything-as-a-source/index.md

```diff
@@ -0,0 +1,374 @@
+    * **403 Access Denied**
+
+3. [Trigger the data import process](/docs/reference/api/xaas):
+
+    Once you have imported all your user profiles to insert or update, start the import processing job to add users to the Okta Universal Directory:
+
+    ```bash
+    curl -i -X PUT \
+      'https://${yourOktaDomain}/api/v1/identity-sources/${identitySourceId}/sessions/${sessionId}/start-import' \
+    -H 'Authorization: SSWS ${apiKey}' \
+    -H 'Content-Type: application/json' 
+    ```
+
+    Possible returned responses:
+    * **200 OK**: The data import process started successfully and returns the following properties:
+
+        ```json
+          {
+            "id": "${sessionId}",
+            "identitySourceId": "${identitySourceId}",
+            "status": "IN_PROGRESS"
+        }
+        ```
+
+    * **400 Bad Request**: Another active Import Session exists for the same identity source.
+    * **403 Access Denied**
+
+4. Monitor the Import Session until the data processing completes
+
+    Request for status of the Import Session to verify that the import job is complete:
+
+    ```bash
+    curl -i -X GET \
+      'https://${yourOktaDomain}/api/v1/identity-sources/${identitySourceId}/sessions/${sessionId} \
+    -H 'Authorization: SSWS ${apiKey}' \
+    -H 'Content-Type: application/json' 
+    ```
+
+    Alternatively, use the [Import Monitoring](https://help.okta.com/okta_help.htm?id=ext-view-import-monitoring-dashboard) page in the Admin Console.
+
+    Possible returned responses:
+    * **200 OK**: The data import process started successfully and returns the following properties:
+
+        ```json
+        {
+            "id": "${sessionId}",
+            "identitySourceId": "${identitySourceId}",
+            "status": "COMPLETED"
+        }
+        ```
+
+    * **400 Bad Request**: Unknown Import Session ID
+    * **403 Access Denied**
+
+5. [Close the Import Session](/docs/reference/api/xaas/)
+
+    Use the DELETE operation to close the Import Session and delete all the bulk user data associated with the session.
+
+    ```bash
+    curl -i -X DELETE \
+      'https://${yourOktaDomain}/api/v1/identity-sources/${identitySourceId}/sessions/${sessionId} \
+    -H 'Authorization: SSWS ${apiKey}' \
+    -H 'Content-Type: application/json' 
+    ```
+
+    Possible returned responses:
+    * **204 No content**: The Import Session closed successfully and contains the `CANCELLED` status. All bulk user data is removed from the session.
+    * **400 Bad Request**: Unknown Import Session ID
+    * **403 Access Denied**
+
+### Bulk deactivate user data
+
+When users have been deactivated or deleted from your HR source, you need to reflect that status in Okta. Okta does not delete user profile objects, it deactivates the users that are no longer active. Use these steps to deactivate a set of user data profiles from Okta.  
+
+1. [Create an Import Session](/docs/reference/api/xaas/):
+
+   ```bash
```

Comment:
Client needs to use /bulk-delete API for this.

Reply:
Yes, it's in the next step. I'm repeating the create an import session operation for this flow.